src/createStore.ts

```diff
@@ -39,35 +39,54 @@ import { kindOf } from './utils/kindOf'
  * @returns A Redux store that lets you read the state, dispatch actions
  * and subscribe to changes.
  */
+
+// 3 overloaded functions
+// 1st overload
```

Comment:
I'm glad @sbroweleit-insight already introduced all that overload stuff in the first PR! Makes gronking this a bit easier!